### PR TITLE
Update o365-data-locations.md

### PR DESCRIPTION
**Pull request by msbemba**
Added FAQ's pertaining data location for shifts  To do and bookings.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2968

**Comment by JoeDavies-MSFT** [comment]
@msbemba Please work with Dean Wierman when requesting changes to this article. Thanks

**Comment by msbemba** [comment]
Thank you @JoeDavies-MSFT

**Comment by msbemba** [comment]
@deanwe can you help us with modifying this document?

**Comment by yogkumgit** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/o365-data-locations.md [hunk 1]
```diff
@@ -79,6 +79,18 @@ Microsoft 365 uses service-side technologies that encrypt customer data at rest
 
 Please review the [Products available by region](https://go.microsoft.com/fwlink/p/?linkid=2093451) page to find data residency information for Microsoft Azure.
 
+### Where can I find data residency information for MIcrosoft Bookings?
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/o365-data-locations.md]:
```suggestion
### Where can I find data residency information for Microsoft Bookings?
```

File: microsoft-365/enterprise/o365-data-locations.md [hunk 2]
```diff
@@ -79,6 +79,18 @@ Microsoft 365 uses service-side technologies that encrypt customer data at rest
...
+
+All customer content is stored in the Exchange Online mailbox that hosts the organization's Bookings.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/o365-data-locations.md]:
```suggestion
All customer content is stored in the Exchange Online mailbox that hosts the organization's Microsoft Bookings.
```

File: microsoft-365/enterprise/o365-data-locations.md [hunk 3]
```diff
@@ -79,6 +79,18 @@ Microsoft 365 uses service-side technologies that encrypt customer data at rest
...
+
+### Where can I find data residency information for MIcrosoft Shifts?
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/o365-data-locations.md]:
```suggestion
### Where can I find data residency information for Microsoft Shifts?
```

File: microsoft-365/enterprise/o365-data-locations.md [hunk 4]
```diff
@@ -79,6 +79,18 @@ Microsoft 365 uses service-side technologies that encrypt customer data at rest
...
+
+Microsoft Shift is a native app of MS Teams so this adheres to the MS team's data location.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/o365-data-locations.md]:
```suggestion
Microsoft Shifts is a native app of Microsoft Teams so this adheres to the data location of Microsoft Teams.
```

File: microsoft-365/enterprise/o365-data-locations.md [hunk 5]
```diff
@@ -79,6 +79,18 @@ Microsoft 365 uses service-side technologies that encrypt customer data at rest
...
+
+### Where can I find data residency information for Microsoft To Do?
+
+Microsoft To do Data is stored as tasks in your Exchange Online mailbox, which also hosts data from other Exchange modules such as emails, events, contacts, and/or notes.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/o365-data-locations.md]:
```suggestion
Microsoft To Do data is stored as tasks in your Exchange Online mailbox, which also hosts data from other Exchange modules such as emails, events, contacts, and/or notes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dmwmsft Please merge this PR. Thanks.

**Comment by JoeDavies-MSFT** [comment]
Closing this pull request. This page is maintained by internal Microsoft processes and is not available for external editing. Thanks

**Pull request closed without merging**